frontend/sozisel-app/src/graphql/queries/session.graphql

```diff
@@ -0,0 +1,40 @@
+query SearchSessions($input: SearchSessionsInput!){
+    searchSessions(input: $input){
```

Comment:
Do you really need all that data that you query in here? Select only data that is being displayed.